tensorflow/contrib/cmake/CMakeLists.txt

```diff
@@ -181,9 +181,14 @@ if (tensorflow_WIN_CPU_SIMD_OPTIONS)
       add_definitions(-DINTEL_MKL_ML)
     endif()
   endif()
-  CHECK_CXX_COMPILER_FLAG("-fopenmp" COMPILER_OPT_OPENMP_SUPPORT)
-  if (COMPILER_OPT_OPENMP_SUPPORT)
+  CHECK_CXX_COMPILER_FLAG("-fopenmp" COMPILER_OPT_OPENMP_SUPPORT1)
```

Comment:
Instead of 1/2 let's use something more explicit.
Let's do GCC_OPENMP_SUPPORT and MSVC_OPENMP_SUPPORT.

And I would simplify the logic like:
```
CHECK_CXX_COMPILER_FLAG("-fopenmp" GCC_OPENMP_SUPPORT)
if (GCC_OPENMP_SUPPORT)
  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fopenmp")
endif()

CHECK_CXX_COMPILER_FLAG("/openmp" MSVC_OPENMP_SUPPORT)
if (MSVC_OPENMP_SUPPORT)
  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /openmp")
endif()
```

Reply:
suggested changes made